dashboard/config/foorm/forms/surveys/pd/summer_workshop_daily_survey.0.json

```diff
@@ -0,0 +1,66 @@
+{
+ "title": "converted from jotform",
+ "pages": [
+  {
+   "name": "page 1",
+   "elements": [
+    {
+     "type": "html",
+     "name": "environment131",
+     "html": "<p><span style=\"font-size: 18pt;\"><strong><span style=\"font-family: arial, helvetica, sans-serif;\">{workshop_course} Summer Workshop - Day {day}</span></strong></span></p>\n<p><span style=\"font-family: arial, helvetica, sans-serif; font-size: 12pt;\">We're eager to hear how you're doing and what your facilitator(s) can do to improve, so we ask that you please take 5-10 minutes to complete this survey. Your feedback will be shared with your facilitator(s), {regional_partner_name} and the Code.org staff.</span></p>\n<p><span style=\"font-family: arial, helvetica, sans-serif; font-size: 12pt;\">Thank you for your time and honesty!</span></p>"
```

Comment:
do you want a variable called `day`?

Reply:
I do.  It's a nice to have for the survey presentation.